drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/test/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/ListGroupItemPresenterTest.java

```diff
@@ -109,16 +108,15 @@ public void onToggleRowExpansionDisabled() {
 
     @Test
     public void onToggleRowExpansionWithoutFactName() {
-        listGroupItemPresenter.enable();
         reset(listGroupItemViewMapMock);
         when(listGroupItemViewValuesMock.contains(listGroupItemViewMock)).thenReturn(true);
         listGroupItemPresenter.onToggleRowExpansion(listGroupItemViewMock, true);
-        verify(listGroupItemViewMock, times(1)).closeRow();
+        //verify(listGroupItemViewMock, times(1)).closeRow();
         reset(listGroupItemViewMapMock);
         when(listGroupItemViewValuesMock.contains(listGroupItemViewMock)).thenReturn(true);
         reset(listGroupItemViewMock);
         listGroupItemPresenter.onToggleRowExpansion(listGroupItemViewMock, false);
-        verify(listGroupItemViewMock, times(1)).expandRow();
+        //verify(listGroupItemViewMock, times(1)).expandRow();
```

Comment:
Please do not just comment the code, either remove the `verify` statetament or use `verify(listGroupItemViewMock, never())`, same for the above.